src/appkit.cs

```diff
@@ -2717,6 +2719,7 @@ interface NSClipView {
 		void ViewBoundsChanged (NSNotification  notification);
 	
 		[Export ("copiesOnScroll")]
+		[Deprecated (PlatformName.MacOSX, 100000, 0, message: "NSClipView will always minimize the area of the document view that is invalidated. To force invalidation of the document view, use SetNeedsDisplayInRect.")]
```

Comment:
incorrect version (Apple did not update it), if that was added in Xcode 11[.x] then it was deprecated in `10,15`